### PR TITLE
update the label of roleOnCase field to 'Role On Case'

### PR DESCRIPTION
**Pull request by aktaskaan**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CCD-4925

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: repository/src/main/resources/db/migration/V20211220_12979__RDM-12979__FlagsFlagDetails.sql [hunk 1]
```diff
@@ -97,7 +97,7 @@ and base_field_type_id is null))
 );
 
 insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
-values ('roleOnCase', 'Flag Type', 'PUBLIC',
+values ('roleOnCase', 'Role On Case', 'PUBLIC',
```
Comment (RebeccaBaker) [inline repository/src/main/resources/db/migration/V20211220_12979__RDM-12979__FlagsFlagDetails.sql:100]: Changing an existing sql file will cause deployed envs to get an error saying this script has already been run but has been modified since and cant be re run. Can we revert this change and stick with the new script please?

Comment (aktaskaan) [reply repository/src/main/resources/db/migration/V20211220_12979__RDM-12979__FlagsFlagDetails.sql]: Thanks Rebecca, it is ready 👍
